import_activity_events.py

```diff
@@ -11,10 +11,12 @@
     service VARCHAR(40) ENCODE zstd,
     ua_browser VARCHAR(40) ENCODE zstd,
     ua_version VARCHAR(40) ENCODE zstd,
-    ua_os VARCHAR(40) ENCODE zstd
+    ua_os VARCHAR(40) ENCODE zstd,
+    country VARCHAR(40) ENCODE zstd,
+    region VARCHAR(40) ENCODE zstd
```

Comment:
do we need to worry about anything else besides `county` and `region` fields, is there more data we get from the geo response?